extensions/wikia/FeedsAndPosts/FeedsAndPostsController.class.php

```diff
@@ -38,10 +38,20 @@ public function getWikiVariables() {
 	}
 
 	public function getArticleData() {
-		$articleId = intval( $this->getRequiredParam( 'id' ) );
+		$articleId = intval( $this->getVal( 'id', null ) );
```

Comment:
If `id`  won't end up being used I think we should remove it—afaik this endpoint will only be used by the tags page.

Reply:
yeah, it makes sense, thanks!